suite-native/atoms/src/Accordion.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import { Pressable, View } from 'react-native';
+import Animated, {
+    Easing,
+    useAnimatedStyle,
+    useSharedValue,
+    withTiming,
+} from 'react-native-reanimated';
+
+import { Icon } from '@trezor/icons';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { Box } from './Box';
+import { Text } from './Text';
+
+type AccordionProps = {
+    title: string;
+    content: string;
+};
+
+const triggerStyle = prepareNativeStyle(utils => ({
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    paddingVertical: utils.spacings.small,
+}));
+
+const ANIMATION_DURATION = 500;
+
+export const Accordion = ({ title, content }: AccordionProps) => {
```

Comment:
nit: I would name this AccordingItem since it's only for one item.

Reply:
[a6215da](https://github.com/trezor/trezor-suite/pull/7792/commits/a6215daba66e77366d8c8b289486867f08807ada)